content/docs/ref/pr.md

```diff
@@ -3,6 +3,59 @@
 Commit specified files to a new branch and create a pull request. If sending a
 report afterwards, consider using `cml send-comment --pr --update`.
 
+ⓘ Pull requests created with `cml pr` **won't** trigger a new CI/CD run
+under any circumstances.
+
+ⓘ You can specify the files to track using any syntax supported by [Git pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec).
+
+## Options
+
+```
+--help                      Show help                                [boolean]
+--version                   Show version number                      [boolean]
+--log                       Maximum log level
+        [string] [choices: "error", "warn", "info", "debug"] [default: "info"]
+--md          Output in markdown format [](url).                    [boolean]
+--remote      Sets git remote.                   [string] [default: "origin"]
+--user-email  Sets git user email.
+                                    [string] [default: "olivaw@iterative.ai"]
+--user-name   Sets git user name.
+                                            [string] [default: "Olivaw[bot]"]
+--repo        Specifies the repo to be used. If not specified is extracted
+              from the CI ENV.                                       [string]
+--token       Personal access token to be used. If not specified in extracted
+              from ENV REPO_TOKEN.                                   [string]
+--driver      If not specify it infers it from the ENV.
+                                       [string] [choices: "github", "gitlab"]
+```
+
```

Comment:
not sure about adding this here...

- auto-generation is being tracked by https://github.com/iterative/cml/issues/361
- seems silly to copy-paste CLI `--help` into a web page
- we only have https://cml.dev/doc/ref/runner#options because it's super complex
  - we really should be replacing the above with proper examples (vis. https://cml.dev/doc/ref/runner#using---cloud-ssh-private) anyway

```suggestion
```

Reply:
Keeping a copy of `cml ··· --help` in the web page [does not](https://dvc.org/doc/command-reference/add#options) look like a [silly](https://dvc.org/doc/command-reference/add#synopsis) idea to me. Still, it would be better to avoid adding additional maintenance burdens to the documentation until we automate https://github.com/iterative/cml/issues/361 properly.

Reply:
> _we only have https://cml.dev/doc/ref/runner#options because it's super complex_

And it would be much better if we didn't have to make an exception for a single command. Either all of them or none of them.

> _we really should be replacing the above with proper examples_ [...] _anyway_

💯 To my mind, both proper examples and an automatically generated command reference.

Reply:
I'm all in favor of automation (https://github.com/iterative/dvc.org/issues/2770). But since it is not available yet, I still prefer having the manual burden of copy-pasting the help command to don't have this info present in the docs, at all. 

It feels even more important to have the automated `help` command on the CML case as I would say that it is more commonly use in C.I./C.D. workflows than in CLI mode (in contrary of DVC), so users might not have easy access to the contents of the command `--help`.